packages/af-shared/src/lib/api/services/permits.ts

```diff
@@ -0,0 +1,18 @@
+import { PersonWorkPermit } from '@/types';
+import apiClient from '../api-client';
+import { TESTBED_API_BASE_URL } from '../endpoints';
+
+export async function getPersonWorkPermits(
+  consentToken?: string
+): Promise<{ permits: PersonWorkPermit[] }> {
+  const { data } = await apiClient.post(
+    `${TESTBED_API_BASE_URL}/testbed/data-product/Permits/WorkPermit_v0.1?source=virtual_finland:development`,
```

Comment:
Tässä `source=virtual_finland:development` -parametri olisi hyvä asentaa jonnekin konffiin että esim. voi eri deploymentissä käyttää eri lähdettä. Tällöin myös se konsentin `dpp://`-tunniste pitäisi muoksia vastaavasti lukemaan tieto samasta konffista. 

Mutta, toisaalta tuo tarve on vähän sellainen nykyisen käyttötapauksen ohi menevä "nice-to-have". Voidaan aivan yhtä hyvin käyttää dev- ja staging-ympäristöille samaa tietolähdettä tässä.

Reply:
Juu kyllä, tässä mentiin nyt hieman rimaa alittaen vielä toistaiseksi, nuo voisi parametrisoida kun/jos tarpeen. 👍 Ja ei vielä tainnut muuta lähdettä tämän yllämainitun lisäksi olla.